src/components/SearchPanel.js

```diff
@@ -18,7 +19,11 @@ export class SearchPanel extends Component {
         windowId={windowId}
         id={id}
         titleControls={<SearchPanelControls cwId={id} windowId={windowId} />}
-      />
+      >
+        <SearchResults
```

Comment:
Don't we need to use the companion window id here too, in case I pop two search windows out?

Reply:
Nice catch. I've pushed that up.